app/experimenter/nimbus-ui/src/components/PageDesign/index.test.tsx

```diff
@@ -25,21 +36,54 @@ describe("PageDesign", () => {
       expect(screen.queryByTestId("PageDesign")).toBeInTheDocument();
     });
   });
+
+  it("redirects to the edit overview page if the experiment status is draft", async () => {
+    const { mock, experiment } = mockExperimentQuery("demo-slug", {
+      status: NimbusExperimentStatus.DRAFT,
+    });
+    mockExperiment = experiment;
+    render(
+      <RouterSlugProvider mocks={[mock]}>
+        <PageDesign />
+      </RouterSlugProvider>,
+    );
+    expect(redirectPath).toEqual("edit/overview");
+  });
+
+  it("redirects to the edit overview page if the experiment status is review", async () => {
+    const { mock, experiment } = mockExperimentQuery("demo-slug", {
+      status: NimbusExperimentStatus.REVIEW,
+    });
+    mockExperiment = experiment;
+    render(
+      <RouterSlugProvider mocks={[mock]}>
+        <PageDesign />
+      </RouterSlugProvider>,
+    );
+    expect(redirectPath).toEqual("edit/overview");
+  });
 });
 
 // Mocking form component because validation is exercised in its own tests.
 jest.mock("../AppLayoutWithExperiment", () => ({
   __esModule: true,
-  default: (props: React.ComponentProps<typeof AppLayoutWithExperiment>) => (
-    <div data-testid="PageDesign">
-      {props.children({
-        experiment: mockExperiment,
-        analysis: mockAnalysisData,
-        review: {
-          isMissingField: () => false,
-          refetch: () => {},
-        },
-      })}
-    </div>
-  ),
+  default: (props: React.ComponentProps<typeof AppLayoutWithExperiment>) => {
+    redirectPath = props.redirect!({
+      status: mockGetStatus(mockExperiment),
+      analysis: mockAnalysisData,
+    });
```

Comment:
In tests where we mock `AppLayoutWithExperiment` we'd need to re-implement a lot to get proper redirection working, so instead I'm just calling the `redirect` function and checking what the response is.